### PR TITLE
Modifies DataServlet.java file

### PR DESCRIPTION
**Pull request by mariah-crawford**
Modifies DataServlet.java file to respond with content that contains my name

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 1]
```diff
@@ -27,6 +27,7 @@
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello world!</h1>");
+    //why doesnt the headers in html format work and only show up as chars?
```
Comment (KidEinstein) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:30]: Shall we remove this comment from here?

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -12,6 +12,44 @@
...
+  // content and not a simple variable.
+  const textPromise = response.text();
+
+  // When the response is converted to text, pass the result into the
+  // addQuoteToDom() function.
+  textPromise.then(addQuoteToDom);
+}
+/** Adds a quote to the DOM. */
+function addQuoteToDom(quote) {
+  console.log('Adding quote to dom: ' + quote);
```
Comment (KidEinstein) [inline portfolio/src/main/webapp/script.js:47]: Great job using logging to capture what's happening!

File: portfolio/src/main/webapp/script.js [hunk 3]
```diff
@@ -12,6 +12,44 @@
...
+
+  // response.text() returns a Promise, because the response is a stream of
+  // content and not a simple variable.
+  const textPromise = response.text();
+
+  // When the response is converted to text, pass the result into the
+  // addQuoteToDom() function.
+  textPromise.then(addQuoteToDom);
+}
+/** Adds a quote to the DOM. */
```
Comment (KidEinstein) [inline portfolio/src/main/webapp/script.js:45]: Nit: Add new line before the function comment.

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java (hunk 1 above)
Comment (mariah-crawford) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:30]: Yeah, I was wondering about that question, do you know why that may be the case that the html format didn't show up properly?

**Pull request closed without merging**